erpnext/payroll/doctype/payroll_settings/payroll_settings.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+# import frappe
+from frappe.model.document import Document
+
+class PayrollSettings(Document):
+	pass
```

Comment:
related code should be moved from hr_settings.py

Reply:
done.